workers/ohsome_quality_analyst/api/request_models.py

```diff
@@ -91,25 +91,30 @@ class BaseBpolys(BaseModel):
         ...,
         title="bpolys",
         example={
-            "type": "Feature",
-            "geometry": {
-                "type": "Polygon",
-                "coordinates": [
-                    [
-                        [8.674092292785645, 49.40427147224242],
-                        [8.695850372314453, 49.40427147224242],
-                        [8.695850372314453, 49.415552187316095],
-                        [8.674092292785645, 49.415552187316095],
-                        [8.674092292785645, 49.40427147224242],
-                    ]
-                ],
-            },
+            "type": "FeatureCollection",
+            "features": [
+                {
+                    "type": "Feature",
+                    "geometry": {
+                        "type": "Polygon",
+                        "coordinates": [
+                            [
+                                [8.674092292785645, 49.40427147224242],
+                                [8.695850372314453, 49.40427147224242],
+                                [8.695850372314453, 49.415552187316095],
+                                [8.674092292785645, 49.415552187316095],
+                                [8.674092292785645, 49.40427147224242],
+                            ]
+                        ],
+                    },
+                }
+            ],
         },
     )
 
     @pydantic.validator("bpolys")
     @classmethod
-    def validate_bpolys(cls, value) -> FeatureCollection | Feature:
+    def validate_bpolys(cls, value) -> FeatureCollection:
         obj = geojson.loads(json.dumps(value))
         validate_geojson(obj)  # Check if exceptions are raised
         return obj
```

Comment:
Check if `obj` is of type `FeatureCollection`. Do not forget to add a test case.